packages/eas-cli/src/fetch.ts

```diff
@@ -8,6 +10,16 @@ export class RequestError extends Error {
   }
 }
 
+function createHttpAgent(): Agent | null {
+  const httpProxyUrl = process.env.http_proxy;
+  if (!httpProxyUrl) {
+    return null;
+  }
+  return createHttpsProxyAgent(httpProxyUrl);
```

Comment:
Does the package validate the URL? If not, maybe it's worth adding custom validation.

Reply:
you will get error like this `[Network] request to https://api.expo.dev/graphql failed, reason: connect ENOENT lksdjflksdjf`

also value here can have a formal `http://username:password@example.com/` not sure if this is valid url